lib/plottr_components/src/components/dialogs/ActsConfigModal.js

```diff
@@ -53,6 +54,10 @@ const ActsConfigModalConnector = (connector) => {
       }
     }
 
+    useEffect(() => {
+      tour.showTour && tourNext()
```

Comment:
What if the user returns to the act config modal while the tour is still running?
Wouldn't that cause it to advance incorrectly?

Reply:
Going to implement a quick fix in React while I look into Finite State Management - in case we want to release a patch in the next day or two with the tour included. Thanks for catching this!